File: BaseLibrary/equip/IEquipment.cs
namespace BaseLibrary.equip {
    /// <summary>
    /// Interface for specified that instance is equipment, can be bought during merchant phase.
    /// </summary>
    public interface IEquipment {
        /// <summary>
        /// How many this equipment cost gold.
        /// </summary>
        int COST { get; }
        
        /// <summary>
        /// This equipment's id for buying.
        /// </summary>
        int ID { get; }
    }
}

File: Java-Client/RobotBattlefield-Client-Java/src/BaseLibrary/equipment/IEquipment.java
package BaseLibrary.equipment;

import BaseLibrary.communication.protocol.IInnerObject;

/**
 * Interface for specified that instance is equipment, can be bought during
 * merchant phase.
 */
public interface IEquipment extends IInnerObject{
	/**
	 * How many this equipment cost gold.
	 */
	public int getCost();

	/**
	 * This equipment's id for buying.
	 */
	public int getID();
}
